CMakeLists.txt

```diff
@@ -1,13 +1,20 @@
 cmake_minimum_required (VERSION 3.0)
 project (libtree)
 
+include(GNUInstallDirs)
+
+set(CACHE BOOL USE_SYSTEM_DEPS OFF)
```

Comment:
Wouldn't it be better to use an option for the user ?

```cmake
option(USE_SYSTEM_DEPS "Use system cppglob" OFF)
```

Reply:
ok